store/db.go

```diff
@@ -0,0 +1,11 @@
+package store
+
+var shortUrlsDb = map[string]string{}
+
+func AddUrl(key string, url string) {
+	shortUrlsDb[key] = url
```

Comment:
we will discuss this later, but it's not safe to insert the link to the map.

is better to use mutex.Lock() and defer mutex.Unlock()

Here is a good example:

https://tour.golang.org/concurrency/9